### ISSUE TITLE
[FEATURE] About limitation

### ISSUE THREAD

**Issue by subversivo58** [gladkikhartem/koshare-router#2]
Hi, @gladkikhartem.

I did not `C++` anything so I did not understand the source.

I wanted to know if there is a limitation of peers (I believe not) but if I want to connect only the first two peers to a channel (like a private room) how do I do this?

I thought of a **"maxpeers"** option next to the subscription since if it is the first peer soon it would have created the channel would not it?

    socket.send(JSON.stringify({"type":msgType.SUBSCRIBE, "topic":"Demo Topic", "maxpeers":2}));

**Comment by gladkikhartem** [comment]
There's no limitation on peers, because it makes 0 sense (if someone else could connect to your room - there's no guarantee that that user is the one you are looking for)

All communications in the room are private by default (i.e. you are sending message directly to peer and no other person sees it, unless you specify message as BROADCAST)

What I did in similar case is I sent a message to all peers currently in a room saying "I support protocol xxx" and the first peer answered me using this protocol was added to global variable, that was used as destination in further communications. 

P.S.
If you want high level of security - consider using WebRTC, something like SimpleWebRTC will work like a charm.
